kolena/_utils/krequests.py

```diff
@@ -143,21 +144,22 @@ def delete(endpoint_path: str, api_version: int = DEFAULT_API_VERSION, **kwargs:
 def raise_for_status(response: requests.Response) -> None:
     if response.status_code == HTTPStatus.UNAUTHORIZED:
         # HTTP 401 is "unauthorized" but used as "unauthenticated"
-        raise UnauthenticatedError(response.content)
+        raise UnauthenticatedError(response=response)
```

Comment:
Can you double check the impact of error reporting on this change, for example console output or stack trace? What would it look like before and after?

Reply:
Reverted